src/lib/components/Body/test/MultipleMarks.test.js

```diff
@@ -0,0 +1,92 @@
+import React from "react";
```

Comment:
Is this still needed?

Reply:
Yes removed  most of it in this commit [fded561d0033d10bcc95dcb0f419bf741995d0ee](https://github.com/APMG/amat-react/pull/7/commits/fded561d0033d10bcc95dcb0f419bf741995d0ee)